addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -6,14 +6,16 @@
         <field name="arch" type="xml">
             <form string="Generate Payment Link">
                 <group>
```

Comment:
why does this change belong to this PR ?

Reply:
At the origin the task was related to the widget in this wizard. I will remove it from the PR.

Reply:
maybe make 2 commits then, they may belong together in the PR but should be merged separately

Reply:
Done